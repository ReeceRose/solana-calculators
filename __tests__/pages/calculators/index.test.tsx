import { render } from '@testing-library/react';

import Index from '../../../pages/calculators/index';

test('calculators/index index page matches snapshot', () => {
  const { asFragment } = render(<Index />);
  expect(asFragment()).toMatchSnapshot();
});
