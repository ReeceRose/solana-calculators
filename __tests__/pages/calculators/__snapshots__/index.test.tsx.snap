// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calculators/ index page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex-grow w-full px-4 mt-12 mb-0"
  >
    <div
      class="flex flex-wrap"
    >
      <div
        class="w-full px-4 mx-auto lg:w-10/12"
      >
        <div
          class="relative flex flex-col w-full min-w-0 break-words bg-gray-700 border-0 rounded-lg shadow-lg"
        >
          <div
            class="p-6 bg-gray-900 rounded-t"
          >
            <div
              class="flex justify-between text-center"
            >
              <h6
                class="text-xl font-bold text-solana-green"
              >
                Solana Utilities
              </h6>
            </div>
          </div>
          <div
            class="flex-auto px-4 lg:px-10"
          >
            <hr
              class="my-6 border-gray-50 border-b-1"
            />
            <h6
              class="text-sm font-bold uppercase text-gray-50"
            >
              Calculators
            </h6>
          </div>
          <div
            class="flex-auto px-4 lg:px-10"
          >
            <hr
              class="my-6 border-gray-50 border-b-1"
            />
            <div
              class="w-full px-4 "
            >
              <p
                class="block mb-2 text-xs font-bold uppercase text-gray-50"
              >
                <a
                  class="text-solana-green hover:text-solana-green-hover"
                  href="/calculators/rent-calculator"
                >
                  Rent Calculator - 
                  <span
                    class="text-gray-50"
                  >
                     Calculate rent cost
                  </span>
                </a>
              </p>
            </div>
          </div>
          <div
            class="mb-6"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
